### PR TITLE
Adguard add support for dns query

### PR DESCRIPTION
**Pull request by Kristof-Mattei**
Allow `/dns-query` on AdGuard for DoH.

### CODE REVIEW

File: adguard.subdomain.conf.sample [hunk 1]
```diff
@@ -47,4 +47,16 @@ server {
         proxy_pass $upstream_proto://$upstream_app:$upstream_port;
 
     }
+    
+    location /dns-query {
+        # to properly use this please set `allow_unencrypted_doh: true` and `force_https: false` in adguard
+        # see https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#configuration-file
+        include /config/nginx/proxy.conf;
+        resolver 127.0.0.11 valid=30s;
```
Comment (nemchik) [inline adguard.subdomain.conf.sample:55]:
```suggestion
        include /config/nginx/resolver.conf;
```
This is the new standard for resolver configurations. Please use this.

**Review by nemchik — CHANGES_REQUESTED** [review]

File: adguard.subdomain.conf.sample (hunk 1 above)
Comment (Kristof-Mattei) [reply adguard.subdomain.conf.sample:55]: Will do. Also do you want me to update the timestamp? Or is that done at release time?

Comment (nemchik) [reply adguard.subdomain.conf.sample:55]: Yes please update the date at the top of the file.

**Comment by Kristof-Mattei** [comment]
@nemchik done. Let me know if anything else needs changes.

**Review by nemchik — APPROVED** [review]

**Pull request merged**